File: TransCoder_tokenized_test_set_functions/cpp/MINIMUM_NUMBER_OF_MANIPULATIONS_REQUIRED_TO_MAKE_TWO_STRINGS_ANAGRAM_WITHOUT_DELETION_OF_CHARACTER.cpp
int countManipulations ( string s1, string s2 ) {
  int count = 0;
  int char_count [ 26 ];
  for ( int i = 0;
  i < 26;
  i ++ ) {
    char_count [ i ] = 0;
  }
  for ( int i = 0;
  i < s1 . length ( );
  i ++ ) char_count [ s1 [ i ] - 'a' ] ++;
  for ( int i = 0;
  i < s2 . length ( );
  i ++ ) {
    char_count [ s2 [ i ] - 'a' ] --;
    if ( char_count [ s2 [ i ] - 'a' ] < 0 ) count ++;
  }
  return count;
}

File: TransCoder_tokenized_test_set_functions/python/MINIMUM_NUMBER_OF_MANIPULATIONS_REQUIRED_TO_MAKE_TWO_STRINGS_ANAGRAM_WITHOUT_DELETION_OF_CHARACTER.py
def countManipulations ( s1, s2 ) :
    count = 0
    char_count = [ 0 ] * 26
    for i in range ( 26 ) :
        char_count [ i ] = 0
    for i in range ( len ( s1 ) ) :
        char_count [ ord ( s1 [ i ] ) - ord ( 'a' ) ] += 1
    for i in range ( len ( s2 ) ) :
        char_count [ ord ( s2 [ i ] ) - ord ( 'a' ) ] -= 1
        if ( char_count [ ord ( s2 [ i ] ) - ord ( 'a' ) ] < 0 ) :
            count += 1
    return count
